src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,42 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+	private static final int _ARRAY_SIZE = 12;
+    private static final int _MAX_ARRAY_INDEX = _ARRAY_SIZE - 1;
+    private static final int _EMPTY_ARRAY_INDICATOR = -1;
+    private static final int _EMPTY_ARRAY_VALUE = -1;
```

Comment:
Original project formatting convention used tabs (should not be changed to spaces). It makes tracking actual changes more difficult.